src/components/Shared/LinePlot/LinePlotComponent.tsx

```diff
@@ -1024,12 +1024,14 @@ export class LinePlotComponent extends React.Component<LinePlotComponentProps> {
                     </Layer>
                 </Stage>
                 }
+                {(this.props.data !== undefined || (this.props.multiPlotPropsMap && this.props.multiPlotPropsMap.size > 0)) &&
```

Comment:
can use optional chaining for simpler code 
`this.props.multiPlotPropsMap?.size > 0`